docs/src/index.md

```diff
@@ -8,6 +8,76 @@ For more illustrations of its usage in conjunction with other packages, the
+    - [`statsmodels.jl`](https://juliastats.org/StatsModels.jl/stable/): For converting heterogeneous DataFrames into homogenous matrices for use with linear algebra libraries or machine learning applications that don't directly support DataFrames. Will do things like convert categorical variables into indicators/one-hot-encodings, create interaction terms, etc.
+    - [`MultivariateStats.jl`](https://multivariatestatsjl.readthedocs.io/en/stable/index.html): linear regression, ridge regression, PCA, component analyses tools. Not well integrated with DataFrames, but easily used using `statsmodels.jl`.
+    - [`Clustering.jl`](https://juliastats.org/Clustering.jl/stable/): All forms of clustering algorithms and tools for cluster evaluation.
+    - [`flux.jl`](https://fluxml.ai/Flux.jl/stable/): A pure-Julia package for building your own machine learning models (a little more low-level than `scikit-learn`). Not well integrated with DataFrames, but easily used using `statsmodels.jl`.
+    - [`scikitlearn.jl`](https://scikitlearnjl.readthedocs.io/en/latest/): A Julia wrapper around the full Python `scikit-learn` machine learning library. Not well integrated with DataFrames, but easily used using `statsmodels.jl`.
+- **Plotting**
+    - [`Plots.jl`](http://docs.juliaplots.org/latest/): Powerful, modern plotting library with a syntax akin to that of `matplotlib` (in Python) or `plot` (in R). [`StatsPlots`](http://docs.juliaplots.org/latest/tutorial/#Using-Plot-Recipes-1) provides `Plots.jl` with recipes for many standard statistical plots.
+    - [`gadfly.jl`]: High-level plotting library with a syntax akin to that of `ggplot` (in R).
+    - [`vega-lite.jl`]: High-level plotting library with a syntax akin to that of `altair` (in Python).
+- **Data Wrangling**:
+    - [`DataFramesMeta.jl`](https://github.com/JuliaData/DataFramesMeta.jl): A range of convenience functions for DataFrames that augment `select` and `transform` to provide a user experience similar to that provided by `dplyr` in R.
+    - [`Query.jl`](https://github.com/queryverse/Query.jl): Convenience functions with a similar goal to `DataFramesMeta.jl`, but built to work with any tabular library in Julia (more on other tabular libraries below).
+- **And More!**
+    - [`lightgraphs.jl`](https://github.com/JuliaGraphs/LightGraphs.jl): A pure-Julia, high performance network analysis library. Edgelists in DataFrames can be easily converted into Graphs using the [`GraphDataFrameBridge.jl`](https://github.com/JuliaGraphs/GraphDataFrameBridge.jl) package.
+- **IO**:
+    - DataFrames work well with a range of formats, including CSVs (using [`CSV.jl`](https://github.com/JuliaData/CSV.jl)), Stata, SPSS, and SAS files (using [`StatFiles`](https://github.com/queryverse/StatFiles.jl)), and reading (though not writing) parquet files (using [`ParquetFiles`](https://github.com/queryverse/ParquetFiles.jl)).
```

Comment:
There are probably more, e.g. see here http://www.david-anthoff.com/jl4ds/stable/fileio/.

Reply:
Any more you had in mind? Definitely need to add [JSON](https://github.com/JuliaData/JSONTables.jl). 

I also figure this will eventually point to an I/O section, we just don't have much of one in the docs yet...